beacon-chain/rpc/eth/beacon/handlers.go

```diff
@@ -855,6 +858,96 @@ func (bs *Server) GetBlockRoot(w http.ResponseWriter, r *http.Request) {
 	http2.WriteJson(w, response)
 }
 
+// ListCommittees retrieves the committees for the given state at the given epoch.
+// If the requested slot and index are defined, only those committees are returned.
+func (bs *Server) ListCommittees(w http.ResponseWriter, r *http.Request) {
+	ctx, span := trace.StartSpan(r.Context(), "beacon.ListCommittees")
+	defer span.End()
+
+	stateId := mux.Vars(r)["state_id"]
+	if stateId == "" {
+		http2.HandleError(w, "state_id is required in URL params", http.StatusBadRequest)
+		return
+	}
+
+	ok, _, e := shared.UintFromQuery(w, r, "epoch")
+	if !ok {
+		return
+	}
+	ok, _, i := shared.UintFromQuery(w, r, "index")
+	if !ok {
+		return
+	}
+	ok, _, s := shared.UintFromQuery(w, r, "slot")
+	if !ok {
+		return
+	}
+
+	st, err := bs.Stater.State(ctx, []byte(stateId))
+	if err != nil {
+		helpers.PrepareStateFetchHTTPError(w, err)
+		return
+	}
+
+	epoch := slots.ToEpoch(st.Slot())
+	if e != 0 {
```

Comment:
This is not correct. In the gRPC version the epoch was a pointer in order to distinguish between no epoch and zero. In your code the state's epoch will be used when someone provides `0`. You should assign the raw epoch returned from `UintFromQuery` to a variable and check against this value.